client.go

```diff
@@ -259,6 +295,8 @@ func (co *Conn) Read(p []byte) (n int, err error) {
 	if err != nil {
 		return n, err
 	}
+
+	co.rtt = time.Since(co.t)
```

Comment:
Which is worth while, but I think the network latency is so huge that we can incur a function call in Code. But fair enough.